modules/tob1/module.json

```diff
@@ -0,0 +1,68 @@
+{
+	"id": "tob1",
+	"name": "tob1",
+	"title": "Tome of Beasts 1",
+	"description": "<p>Whether you need dungeon vermin or world-shaking villains, <em>Tome of Beasts 1</em> has it. This book presents foes suitable for any campaign setting&mdash;from tiny drakes and peculiar spiders to demon lords and ancient dragons.</p><p>These monsters have been designed so that GMs can use them in their favorite settings for fantasy adventure. This monstrous tome contains:</p><ul><li>More than <strong>400 new monsters</strong> for use with the <strong>D&amp;D Beyond Encounters tool</strong>.</li><li>Updated with errata, streamlined mechanics, and new monster art from the original <em>Tome of Beasts</em>.</li><li>Expansive tables that organize creatures by challenge rating, creature type, and terrain.</li></ul><hr />",
```

Comment:
```suggestion
	"description": "<p>Whether you need dungeon vermin or world-shaking villains, <em>Tome of Beasts 1</em> has it. This book presents foes suitable for any campaign setting&mdash;from tiny drakes and peculiar spiders to demon lords and ancient dragons.</p><p>These monsters have been designed so that GMs can use them in their favorite settings for fantasy adventure. This monstrous tome contains:</p><ul><li>More than <strong>400 new monsters</strong> for use with the <strong>D&amp;D Beyond Encounters tool</strong>.</li><li>Updated with errata, streamlined mechanics, and new monster art from the original <em>Tome of Beasts</em>.</li><li>Expansive tables that organize creatures by challenge rating, creature type, and terrain.</li></ul>",
```